views/admin/index.erb

```diff
@@ -49,4 +49,30 @@
     <li><a href="/change-password">Change Password</a></li>
     <li><a href="/multifactor-manage">Manage Multifactor Authentication</a></li>
   </ul>
+
+  <% if @info_pages.any? %>
+    <h2>Info Pages</h2>
+    <table class="info-page-table">
+      <thead>
+        <tr>
+          <th>Summary</th>
+          <th>Related Resources</th>
+          <th>Created At</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @info_pages.each do |info_page| %>
+          <tr>
+            <td><a href="/model/Page/<%= info_page.ubid %>"><%= info_page.summary %></a></td>
+            <% if info_page.details["related_resources"] %>
+              <td><%== linkify_ubids(info_page.details["related_resources"].join(" ")) %></td>
+            <% else %>
+              <td><em>No related resources</em></td>
```

Comment:
Avoid using `<em>`. If you would like this styled a certain way, add a class and style it via the stylesheet.